src/page-modules/assistant/trip/trip-pattern/index.tsx

```diff
@@ -108,15 +118,27 @@ export default function TripPattern({
                     )}
                   </div>
 
-                  <Typo.span textType="body__tertiary">
-                    {formatToClock(leg.expectedStartTime, language, 'floor')}
-                  </Typo.span>
+                  <div className={style.timeStartContainer}>
+                    <Typo.span textType="body__tertiary">
+                      {formatToClock(leg.expectedStartTime, language, 'floor')}
+                    </Typo.span>
+                    {!isEqualOnMinuteLevel(
+                      leg.aimedStartTime,
+                      leg.expectedStartTime,
+                    ) && (
+                      <Typo.span
+                        textType="body__tertiary"
+                        className={style.lineThrough}
+                      >
+                        {formatToClock(leg.aimedStartTime, language, 'floor')}
+                      </Typo.span>
+                    )}
+                  </div>
                 </div>
 
                 {(i < expandedLegs.length - 1 || collapsedLegs.length > 0) && (
                   <div className={style.legs__legLineContainer}>
                     <div className={style.legs__legLine} />
-                    <div className={style.legs__legLine} />
```

Comment:
Was this removed accidentally? There should be two lines according to the figma sketches
![image](https://github.com/AtB-AS/planner-web/assets/43166974/2b891352-c072-4018-b170-88ec6b96d660)

Reply:
Yes, that div was mistakenly deleted🤔